lib/matplotlib/contour.py

```diff
@@ -720,8 +720,6 @@ class ContourSet(cm.ScalarMappable, ContourLabeler):
     %(contour_set_attributes)s
     """
 
-    ax = _api.deprecated("3.3")(property(lambda self: self.axes))
```

Comment:
I'm not sure there was a changelog entry for this removal?

Reply:
Should be here: https://github.com/matplotlib/matplotlib/pull/20447/files#diff-891a63498fc22bcffb7555796d5a46cdda5e432360eaa4b13f459fc029af7858R52-R55